### DISCUSSION TITLE
Lucky::Action.query_param_declarations should include the full declaration

### DISCUSSION THREAD

**Discussion by watzon** [luckyframework/lucky#1796] [Feature requests & Ideas]
It seems like currently the declarations listed in [`Lucky::Action.query_param_declarations`](https://luckyframework.github.io/lucky/Lucky/Action.html#query_param_declarations%3AArray%28String%29-class-method) don't include the full declaration signature, and instead only include the variable's name and type. I'm trying to fix `lucky_swagger` and the one thing I'm missing right now is the ability to show the default value for parameters.

**Comment by jwoertink** [comment]
This happens here https://github.com/luckyframework/lucky/blob/c12e50d80aea583cf7d0cf33b88e547bd85cfa69/src/lucky/routable.cr#L452 which I think is really only used https://github.com/luckyframework/lucky/blob/c12e50d80aea583cf7d0cf33b88e547bd85cfa69/tasks/routes.cr#L34-L37 outside of 3rd party usage. But since it's just a String, my guess is it should be fine to just add the default value if there is one.
